NeurodataWithoutBorders/pynwb
# Validate new specs as they are added to NamespaceBuilder

## Motivation

Fixes #434 where invalid specs were not noticed until load time by validating the spec each time one is added to a builder.

## How to test the behavior?

Attempt to add the same spec to a builder twice ( see the example in #434 ) and get a ValueError on the second call to add_spec.

## Checklist
